biggraphite/drivers/cassandra.py

```diff
@@ -1689,17 +1669,22 @@ def syncdb(self, retentions=None, dry_run=False):
         return schema
 
     @UPDATED_ON.time()
-    def touch_metric(self, metric_name):
+    def touch_metric(self, metric):
         """See the real Accessor for a description."""
-        super(_CassandraAccessor, self).touch_metric(metric_name)
+        super(_CassandraAccessor, self).touch_metric(metric)
 
         if self.__bulkimport:
             return
 
-        self._execute_async_metadata(
-            self.__touch_metrics_metadata_statement,
-            (metric_name,)
-        )
+        if not metric.updated_on:
+            delta = self.__metadata_touch_ttl_sec + 1
+        else:
+            delta = int(time.time()) - int(metric.updated_on / 1000)
```

Comment:
is updated_on a timestamp or a datetime ?

Reply:
the same as in cassandra. Btw, it's just code that moved from touch_metadata_on_need to here (modulo the fact that it's now stored in metric object directly)

Reply:
in cassandra it's a timeuuid :P , but ok if it just moved then it should be fine